services/src/test-library/pom.xml

```diff
@@ -45,6 +45,11 @@
     </dependencyManagement>
 
     <dependencies>
+        <dependency>
+            <groupId>org.openkilda</groupId>
+            <artifactId>grpc-speaker</artifactId>
```

Comment:
Are you sure what need this artifact?

Reply:
Fixed. I left the grpc-api dependency only.